ncl/ncl_entries/climatology.ipynb

```diff
@@ -12,30 +12,37 @@
    "metadata": {},
```

Comment:
<p>While I agree that this is probably what I would expect here, this is a difference from the behavior of the rest of NCL applications.</p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/NCAR/geocat-applications/pull/228/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='1'/>

Reply:
It would be different from some of the more developed NCL applications notebooks (that I think may have modeled the original behavior here 😞 ) though we're not consistent overall at the moment either 🙃 (e.g. the time notebook and spectral analysis are totally different).

I can pull this out into a separate issue, but I do think it's much more helpful this way.  The number of times I unexpectedly went to the NCL docs is somewhat embarrassing. 

The header level changes are also something that'll introduce some inconsistency FWIW.